apps/ideas/views.py

```diff
@@ -106,7 +106,7 @@ def done(self, form_list, **kwargs):
                 if field not in special_fields
                 }
         )
-
+        idea_sketch.co_workers.add(self.request.user)
```

Comment:
Maybe we should add an transaction here, so that we don't create an idea_sketch without coworker accidentally.